R/create.R

```diff
@@ -19,17 +24,22 @@
 #' @param rstudio If `TRUE`, calls [use_rstudio()] to make the new package or
 #'   project into an [RStudio
 #'   Project](https://r-pkgs.org/workflow101.html#sec-workflow101-rstudio-projects).
-#'    If `FALSE` and a non-package project, a sentinel `.here` file is placed so
-#'   that the directory can be recognized as a project by the
-#'   [here](https://here.r-lib.org) or
-#'   [rprojroot](https://rprojroot.r-lib.org) packages.
+#'
+#'   If `FALSE`, the goal is to ensure that the directory can be recognized as
+#'   a project by, for example, the [here](https://here.r-lib.org) package. If
+#'   the project is neither an R package nor a Quarto project, a sentinel
+#'   `.here` file is placed to mark the project root.
```

Comment:
This is mildly aspirational. I have only recently added `rprojroot::is_quarto_project` to usethis's project-finding criteria and I have a PR in flight to have the here package do the same: https://github.com/r-lib/here/pull/130. But I think this will hopefully be true soon anough.